app/account/hooks/useUserUpdate.ts

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react'
+import { useToast } from '@chakra-ui/react'
+import { useRouter } from 'next/navigation'
+import { useUserId } from '@/providers/session-provider'
+import { useUserGet } from '.'
+import { useUpdateUserMutation } from '@generated/api'
+
+export const useUserUpdate = () => {
```

Comment:
This does not fix yet. Could you fix it?

https://github.com/tabi-memo/frontend/pull/31#discussion_r1493426737